src/positions.ts

```diff
@@ -31,8 +31,10 @@ export const findPositionsFromEvents = (options: DOMFunctions) => (
 ): Observable<PositionEvent> =>
     merge(
         from(elements).pipe(
-            switchMap(element => fromEvent<MouseEvent>(element, 'mouseover')),
-            map(event => ({ event, eventType: 'mouseover' as 'mouseover' })),
+            switchMap(element =>
+                merge(fromEvent<MouseEvent>(element, 'mouseover'), fromEvent<MouseEvent>(element, 'mousemove'))
```

Comment:
I think there might be assumptions in codeintellify that this function only emits when a different token is hovered. Please make sure that this does not cause over-triggering requests.

Reply:
Added tests for this general behavior and for the case you mention here. No impl changes required.